### PR TITLE
CHANGELOG: update from #12288 and #12299

### PR DESCRIPTION
**Pull request by jingyih**
CHANGELOG: update from #12288 and #12299

### CODE REVIEW

File: CHANGELOG-3.4.md [hunk 1]
```diff
@@ -9,7 +9,20 @@ The minimum recommended etcd versions to run in **production** are 3.2.28+, 3.3.
 <hr>
 
 
-## v3.4.13 (2020 TBD)
+## v3.4.14 (2020 TBD)
+
+See [code changes](https://github.com/etcd-io/etcd/compare/v3.4.11...v3.4.13) and [v3.4 upgrade guide](https://github.com/etcd-io/etcd/blob/master/Documentation/upgrades/upgrade_3_4.md) for any breaking changes.
```
Comment (spzala) [inline CHANGELOG-3.4.md:14]: Thanks @jingyih LGTM but a nit,  this should be v3.4.14 as TBD right?

Comment (spzala) [reply CHANGELOG-3.4.md:14]: i.e. in here v3.4.11...v3.4.13

Comment (jingyih) [reply CHANGELOG-3.4.md]: Updated. Thanks!

Comment (spzala) [reply CHANGELOG-3.4.md]: Thanks @jingyih

**Pull request merged**